src/Distributions.jl

```diff
@@ -25,6 +25,7 @@ import StatsBase: kurtosis, skewness, entropy, mode, modes,
 import PDMats: dim, PDMat, invquad
 
 using SpecialFunctions
+using Base.MathConstants
```

Comment:
Can we qualify `eulergamma` instead? I guess it could be slightly dangerous/surprising if we load generic symbols such as greek letters gamma and phi in the Distributions module (https://github.com/JuliaLang/julia/blob/8e630552924eac54c809aa7bc30871c7df1582d3/base/mathconstants.jl#L11).
```suggestion
```